docs/en/enterprise-edition/policy-reference/google-cloud-policies/google-cloud-kubernetes-policies/bc-google-cloud-123.adoc

```diff
@@ -0,0 +1,81 @@
+
+== GKE container cluster configuration of NodePools is at the cluster level
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/gcp/GKEDontUseNodePools.py[CKV_GCP_123]
+
+|Severity
+|LOW
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is designed to identify the use of NodePools within the configuration of a Google Kubernetes Engine (GKE) cluster. The rationale behind this check is that utilizing NodePools in the cluster configuration may unnecessarily complicate the management of the cluster.
```

Comment:
This policy checks whether Google Kubernetes Engine (GKE) clusters uses NodePools within their configuration. The reason for this check is that using NodePools in the cluster configuration can unnecessarily complicate cluster management.